packages/fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -190,7 +190,7 @@ export class ForkChoice implements IForkChoice {
       return headBlock;
     }
 
-    const {prelimProposerHead} = this.getPreliminaryProposerHead(headBlock, parentBlock, proposalSlot);
+    const {prelimProposerHead} = this.getPreliminaryProposerHead(headBlock, parentBlock, secFromSlot, proposalSlot);
```

Comment:
this `predictProposerHead` should not need `secFromSlot`, I suppose it should skip `is_proposing_on_time` condition in this case